tests/snuba/sessions/test_sessions.py

```diff
@@ -1165,7 +1165,7 @@ def setUp(self):
         self.another_project = self.create_project()
         self.third_project = self.create_project()
 
-        self.now_dt = datetime(2000, 3, 20, 17, 40, 0)
+        self.now_dt = datetime.now()
```

Comment:
This was deliberately changed to a fixed date in https://github.com/getsentry/sentry/pull/29445, so maybe we can also use `MOCK_DATETIME` here?

```suggestion
        self.now_dt = MOCK_DATETIME
```

Reply:
Well, that means we have to change the constant every 90 days, which is part of what I'm trying to avoid with this change